src/main.js

```diff
@@ -1,11 +1,18 @@
 // Create variables targetting the relevant DOM elements here 👇
+var randomCover = document.querySelector(".random-cover-button");
```

Comment:
Great job on creating these variables! We didn't need these created yet for iteration 0, but nice to have set up for later.